src/simple-card/index.html

```diff
@@ -3,3 +3,4 @@
 </gb-link>
 <gb-link url="{ $product.link() }" on-click="{ $product.onClick }">{ $product.data.title }</gb-link>
 <p>{ $product.data.price }</p>
+<div data-is="gb-{ $product.display || 'swatch' }-variant-controls" field="{ $product.field }"></div>
```

Comment:
we won't include the variant display by default, it will be a matter of implementation to add the desired controls to the `<gb-product>` template

Reply:
so if we have
```html
<gb-product>
 <!-- content -->
</gb-product>
```

then it will only use the main data, but if we have

```html
<gb-product>
  <!-- content -->

  <gb-some-variant-controls field="color" />
<gb-product>
```

then it will switch over the variant data based on the controls